### PR TITLE
Added Move Counter

### PR DESCRIPTION
**Pull request by seyonv**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (KerimovEmil) [inline .idea/workspace.xml:1]: We should add a `.gitignore` file to remove files like this.

File: problem.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ class Problem:
         fib_quotient is ... of quotient  (111)
     """
 
-    def __init__(self, lower=4, upper=10):
+    def __init__(self, lower=10, upper=15):
```
Comment (KerimovEmil) [inline problem.py:22]: did you want to commit this change?

File: .idea/workspace.xml (hunk 1 above)
Comment (KerimovEmil) [reply .idea/workspace.xml:1]: other than this file the PR looks good

Comment (seyonv) [reply .idea/workspace.xml:1]: True True

File: problem.py (hunk 2 above)
Comment (seyonv) [reply problem.py:22]: No, but we can undo this later, probably want to use variables in SETTINGS for this as well. We can later use a structure .yml file for these setting variables so there's an organized hierarchy

**Review by KerimovEmil — APPROVED** [review]

**Pull request merged**